zowe-cli-id-federation-plugin/package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "build": "tsc --pretty && npm run checkTestsCompile && madge -c lib",
     "checkTestsCompile": "echo \"Checking that test source compiles...\" && tsc --project __tests__/test-tsconfig.json --noEmit ",
-    "prebuild": "npm run clean && npm run lint && echo Using TypeScript && tsc --version",
```

Comment:
Why not lint?

Reply:
Because lint called from build.gradle as a dependency to npmBuild task. There was recommendation to have such dependencies in gradle instead of having them in package.json.

Reply:
Ok, I was not sure if it can be useful during the development